src/EFCore/EF.cs

```diff
@@ -55,6 +58,15 @@ public static partial class EF
         [NotParameterized] string propertyName)
         => throw new InvalidOperationException(CoreStrings.PropertyMethodInvoked);
 
+    /// <summary>
+    ///     . 
+    /// </summary>
+    /// <typeparam name="T">.</typeparam>
+    /// <returns>.</returns>
```

Comment:
Can we find out if the below formatting of the code is OK?

I think we can omit `<typeparam>` and `<returns>` here as they're useless...

```suggestion
    /// <summary>
    ///     Used set a property to its default value within <see cref="M:RelationalQueryableExtensions.ExecuteUpdate" /> or
    ///     <see cref="M:RelationalQueryableExtensions.ExecuteUpdateAsync" />.
    /// </summary>
    /// <example>
    ///     <para>
    ///         The following resets all Blog names to their default value:
    ///     </para>
    ///     <code>
    ///         ctx.Blogs.ExecuteUpdate(s => s.SetProperty(b => b.Name, b => EF.Default{string}()));
    ///     </code>
    ///     <para>
    ///         Depending on how the property is configured, this may be <see langword="null" />, or another value defined via
    ///         <see cref="M:RelationalPropertyBuilderExtensions.UasDefaultValue" /> or similar.
    ///     </para>
    /// </example>
```

Reply:
Typeparam is needed because user has to explicitly specify it.

Reply:
`M:` should work.